Tools/Scripts/webkitpy/layout_tests/servers/web_platform_test_server.py

```diff
@@ -45,6 +45,7 @@ def wpt_config_json(port_obj):
         return
     config = json.loads(fs.read_text_file(config_wk_filepath))
     if port_obj.supports_localhost_aliases and not port_obj.get_option('disable_wpt_hostname_aliases'):
+        config['bind_address'] = False  # Don't bind so users can also user localhost/127.0.0.1.
```

Comment:
I don't understand this comment, could you please elaborate?

Reply:
See also the comment at from (@TingPing's change which regressed this): https://github.com/WebKit/WebKit/blob/83e92a3d7ab3fcddacde92972ea1aade639459d3/Tools/Scripts/webkitpy/layout_tests/servers/web_platform_test_server.py#L99-L100

AIUI, the main part of the reason here is that `supports_localhost_aliases` only implies WKTR/DRT can load from those aliases, and it is useful to also be able to load things from other UAs.

Reply:
Thank you. Ideally, the more detailed explanation should be in the code, not just here in a PR comment.

Reply:
Fair enough. I can't say that I fully understand what's going on, but given that no one may fully understand it, what you said seems like a sufficient enough pointer to help future us.

Reply:
I would change the comment slightly.

The reason we avoid binding the address is because we will be using arbitrary addresses like `web-platform.test` and wpt will fail to bind that.

Reply:
> The reason we avoid binding the address is because we will be using arbitrary addresses like `web-platform.test` and wpt will fail to bind that and fatality error.

Ah, if that's the concern then we should just set `server_host` to `127.0.0.1` and everything should be fine. And that's resolved this because… the comment doesn't exist any more.